gobblin-distribution/build.gradle

```diff
@@ -47,6 +47,7 @@ dependencies {
   compile project(':gobblin-yarn')
   compile project(':gobblin-aws')
   compile project(':gobblin-runtime-hadoop')
+  compile project(':gobblin-azure-datalake')
```

Comment:
This should not be part of gobblin-distribution. Instead, add it to `gobblin-distribution/gobblin-flavor-full.gradle`